src/main/java/frc/robot/commands/arm/ArmManual.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.arm;
+
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.math.filter.SlewRateLimiter;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.arm.ArmSubsystem;
+
+/**
+ * This command moves the arm manually.
+ */
+public class ArmManual extends Command {
```

Comment:
The name can be a little more specific. Like `ArmManualSetSpeed`. Ideally, commands should be named as an action.